vllm/core/block/prefix_caching_block.py

```diff
@@ -1,13 +1,13 @@
 """Token blocks."""
 
-from itertools import takewhile
 from os.path import commonprefix
 from typing import Dict, FrozenSet, Iterable, List, Optional, Tuple
 
 from vllm.core.block.common import (CopyOnWriteTracker,
                                     get_all_blocks_recursively)
 from vllm.core.block.interfaces import Block, BlockAllocator, BlockId, Device
-from vllm.core.block.naive_block import NaiveBlock, NaiveBlockAllocator
+from vllm.core.block.naive_block import (BlockPool, NaiveBlock,
```

Comment:
qq: Why import `BlockPool` from `vllm.core.block.naive_block` instead of `vllm.core.block.common`?

Reply:
Forgot to change it. It was originally in naive_block.